lumigator/frontend/src/stores/experiments/store.js

```diff
@@ -165,31 +165,31 @@ export const useExperimentStore = defineStore('experiment', () => {
       return {
         example,
         bertscore: {
-          f1: objectData.bertscore.f1[index],
-          f1_mean: objectData.bertscore.f1_mean,
-          hashcode: objectData.bertscore.hashcode,
-          precision: objectData.bertscore.precision[index],
-          precision_mean: objectData.bertscore.precision_mean,
-          recall: objectData.bertscore.recall[index],
-          recall_mean: objectData.bertscore.recall_mean,
+          f1: objectData.bertscore?.f1?.[index] ?? 0,
+          f1_mean: objectData.bertscore?.f1_mean ?? 0,
+          hashcode: objectData.bertscore?.hashcode ?? 0,
+          precision: objectData.bertscore?.precision?.[index] ?? 0,
+          precision_mean: objectData.bertscore?.precision_mean ?? 0,
+          recall: objectData.bertscore?.recall?.[index] ?? 0,
+          recall_mean: objectData.bertscore?.recall_mean ?? 0,
         },
-        evaluation_time: objectData.evaluation_time,
-        ground_truth: objectData.ground_truth[index],
+        evaluation_time: objectData.evaluation_time ?? 0,
+        ground_truth: objectData.ground_truth?.[index],
         meteor: {
-          meteor: objectData.meteor.meteor[index],
-          meteor_mean: objectData.meteor.meteor_mean,
+          meteor: objectData.meteor?.meteor?.[index] ?? 0,
+          meteor_mean: objectData.meteor?.meteor_mean ?? 0,
         },
         model: objectData.model,
-        predictions: objectData.predictions[index],
+        predictions: objectData.predictions?.[index],
         rouge: {
-          rouge1: objectData.rouge.rouge1[index],
-          rouge1_mean: objectData.rouge.rouge1_mean,
-          rouge2: objectData.rouge.rouge2[index],
-          rouge2_mean: objectData.rouge.rouge2_mean,
-          rougeL: objectData.rouge.rougeL[index],
-          rougeL_mean: objectData.rouge.rougeL_mean,
-          rougeLsum: objectData.rouge.rougeLsum[index],
-          rougeLsum_mean: objectData.rouge.rougeLsum_mean,
+          rouge1: objectData.rouge?.rouge1?.[index] ?? 0,
+          rouge1_mean: objectData.rouge?.rouge1_mean ?? 0,
+          rouge2: objectData.rouge?.rouge2?.[index] ?? 0,
+          rouge2_mean: objectData.rouge?.rouge2_mean ?? 0,
+          rougeL: objectData.rouge?.rougeL?.[index] ?? 0,
+          rougeL_mean: objectData.rouge?.rougeL_mean ?? 0,
+          rougeLsum: objectData.rouge?.rougeLsum?.[index] ?? 0,
+          rougeLsum_mean: objectData.rouge?.rougeLsum_mean ?? 0,
```

Comment:
This was me, it's something we need to chat through to figure out the right way to solve it.